compose/init-db.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    SELECT 'CREATE DATABASE rest'
+    WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'rest')\gexec
```

Comment:
This is so we can have one postgres in the compose file, and use it for SQ + REST api.

Without this file you would have to manually make the DB. Its mapped to be in an "auto run" dir in the container